tests/test_kmeans_segmentation.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+import numpy as np
+from Segmentation.kmeans_segmentation import segment_blue_nuclei
+from data_types import SegmentedImage
+
+
+class TestShapeFeatures(unittest.TestCase):
+    def setUp(self):
+        self.image = np.array(
+            [[[193., 11., 209.], [193., 114., 209.], [255., 255., 255.]],
+             [[255., 215., 255.], [68., 54., 154.], [255., 255., 255.]],
+             [[255., 251., 255.], [158, 54., 154.], [68., 233., 124.]]])
+        self.segmented = segment_blue_nuclei(self.image)
+
+    def test_if_returns_correct_type(self):
```

Comment:
I belive that checking for the correct return type is not sufficient, because the _SegmentedImage_ class has very few assertions and practically checks nothing about the segmented image. Maybe add more tests here that actually check if the segmentation went as expected, or add more assertions to the `__init__` method of the _SegmenedImage_ class.

Reply:
The problem here is that it is pretty hard to test segmentation, I tried to do that, but my original plan to test this function didn't work as expected. I will try to figure that out.